src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/PaymentMessageProcessorTest.java

```diff
@@ -125,7 +126,8 @@ public void should_complete_the_message_when_processing_is_successful() throws E
         // given
         IMessage validMessage = getValidMessage();
         given(messageReceiver.receive()).willReturn(validMessage);
-        willReturn(paymentMessage(CCD_CASE_NUMBER, IS_EXCEPTION_RECORD)).given(paymentMessageParser).parse(any());
+        willReturn(paymentMessage(CCD_CASE_NUMBER, IS_EXCEPTION_RECORD))
+            .given(paymentMessageParser).parse(any(), any());
```

Comment:
not making all those formatting changes will make it easier to review actual changes